code-challenges.js

```diff
@@ -47,3 +83,6 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
```

Comment:
Please push any code you have for this problem!

Reply:
i will do so as soon thank you